sendsmaily_subscribe/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM drupal:7-apache
+RUN apt-get update && apt-get install -y mysql-client libnotify-bin
+RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN composer global require drush/drush:8.* \
+  && rm -f /usr/local/bin/drush \
+  && ln -s ~/.composer/vendor/bin/drush /usr/local/bin/drush \
+  && drush core-status -y \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
It would make more sense if `/var/lib/apt/lists/` clean up is next to `apt-get` commands. Is there a reason why you keep it here?

Reply:
No reason, I think it was like so in another docker file